src/ServerError.js

```diff
@@ -0,0 +1,26 @@
+import Parse from 'parse';
+
+/**
+ * Constructs a new ServerError object with the given message and source.
+ *
+ * The message is an error that is intended for the server logs
+ * while the source message is intended for the server logs to aid a developer
+ *
+ * @class ServerError
+ * @constructor
+ * @param {ErrorCode} code the Parse.Error code
+ * @param {String} message will be sent to clients
+ * @param {String} serverErrorLog additional information
+ *                  about the error for logging on the server
+ * @param {String} stack the stack trace
+ */
+export default class ServerError extends Parse.Error {
```

Comment:
nope:

  console.log(err.constructor == ServerError);
false
  console.log(err.constructor == Error);
true

something is futzing with my object.  I'd guess express.  I made a half assed attempt to figure out what was causing it in the express source code...no joy.